### PR TITLE
Mzouink

### PR DESCRIPTION
**Pull request by rhoadesScholar**

### CODE REVIEW

File: dacapo/experiments/datasplits/datasets/arrays/concat_array.py [hunk 1]
```diff
@@ -93,6 +95,7 @@ def num_channels(self):
         return len(self.channels)
 
     def __getitem__(self, roi: Roi) -> np.ndarray:
+        logger.info(f"Concat Array: Get Item {self.name} {roi}")
```
Comment (rhoadesScholar) [inline dacapo/experiments/datasplits/datasets/arrays/concat_array.py]:
```remove
        logger.info(f"Concat Array: Get Item {self.name} {roi}")
```

File: dacapo/experiments/datasplits/datasets/arrays/concat_array.py [hunk 2]
```diff
@@ -116,5 +119,6 @@ def __getitem__(self, roi: Roi) -> np.ndarray:
             axis=0,
         )
         if concatenated.shape[0] == 1:
-            raise Exception(f"{concatenated.shape}, shapes")
+            logger.info(f"Concatenated array has only one channel: {self.name} {concatenated.shape}")
+            # raise Exception(f"{concatenated.shape}, shapes")
```
Comment (rhoadesScholar) [inline dacapo/experiments/datasplits/datasets/arrays/concat_array.py]: remove

**Comment by rhoadesScholar** [comment]
@mzouink Are we still planning to try to merge this, or can we safely delete the branch?

**Comment by mzouink** [comment]
we still need to pull attention unet from the branch then we can delete it

**Comment by rhoadesScholar** [comment]
Attention UNet pulled! Closing PR and deleting branch.

**Pull request closed without merging**